### PR TITLE
Initial project skeleton

### PR DESCRIPTION
**Pull request by andrej-khatkevich**

### CODE REVIEW

File: RubberDucksTest/src/test/java/mavenizer/staticPO/Browser.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package mavenizer.staticPO;
+
+public enum Browser {
```
Comment (Fulcrum81) [inline RubberDucksTest/src/test/java/mavenizer/staticPO/Browser.java:3]: This class can also be moved to the helpers package

File: RubberDucksTest/src/test/java/mavenizer/staticPO/TestBase.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package mavenizer.staticPO;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class TestBase {
```
Comment (Fulcrum81) [inline RubberDucksTest/src/test/java/mavenizer/staticPO/TestBase.java:10]: this class should probably go to the mavenizer root package, it doesn't really belong in staticPO

**Review by Fulcrum81 — CHANGES_REQUESTED** [review]
Good job overall, please correct the 2 comments and I'll approve

File: RubberDucksTest/src/test/java/mavenizer/staticPO/TestBase.java (hunk 2 above)
Comment (andrej-khatkevich) [reply RubberDucksTest/src/test/java/mavenizer/staticPO/TestBase.java]: Thanks for hint!

**Comment by andrej-khatkevich** [comment]
Thanks

**Pull request closed without merging**